Validation/RecoEgamma/python/egammaValidationMiniAOD_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms

```

Comment:
why this file is needed?
it seems to me that the electron and photon validation stay separate in the end in terms of config files.